example/package.json

```diff
@@ -12,29 +12,32 @@
     "test": "jest"
   },
   "dependencies": {
-    "@expo/vector-icons": "^10.2.0",
-    "@react-native-community/datetimepicker": "^2.4.0",
-    "@react-native-community/masked-view": "^0.1.10",
+    "@expo/vector-icons": "^10.0.0",
```

Comment:
any reason why you downgraded this?

did you happen to run this with `expo install` for sdk 38 so it installed the right versions for that package?

Reply:
Just ran expo install on whole project to ensure proper versions of all packages, all went well